src/discussions/posts/data/thunks.js

```diff
@@ -67,7 +106,8 @@ export function fetchThreads(courseId, {
     try {
       dispatch(fetchThreadsRequest({ courseId }));
       const data = await getThreads(courseId, options);
-      dispatch(fetchThreadsSuccess(camelCaseObject(data)));
+      const normalised = normaliseThreads(camelCaseObject(data));
```

Comment:
just a suggestion to rename this variable named **normalised** to **normalisedData**? maybe or if you have any more accurate name in mind?